apps/omg_db/test/omg_db/db_test.exs

```diff
@@ -27,6 +27,7 @@ defmodule OMG.DBTest do
 
```

Comment:
commenting on the `@moduledoc` :point_up:  - will this become a test agnostic of the driver/backend (the moduledoc says `LevelDB` support). Maybe the wording can be updated there?